content/composer/overview.md

```diff
@@ -39,34 +43,37 @@ right side of the screen.
 
 ![Blueprint Composer topology]({{< img "ui/composer/sidepane.png" >}})
 
-# Inputs & Outputs
-The inputs & outputs page contains an option to add inputs and outputs to a blueprint.yaml. The ‘Name’ field is mandatory and must be unique, description is optional, and the value could be filled or be left empty.
+### Inputs & Outputs tab
+The inputs & outputs page contains an option to add inputs and outputs to a blueprint.yaml. 
+The ‘Name’ field is mandatory and must be unique, description and value are optional.  <br />
 
-To add another field, click the ‘+’ button on the right. To delete an entered field, press the trashcan icon.
+To add another field, click the ‘+’ button on the right. 
+To delete an entered field, press the trashcan icon.
 
 ![Blueprint Composer inputs]({{< img "ui/composer/inputs-outputs.png" >}})
 
-# Definitions
+### Definitions tab
 In the definitions section you can define new inline types, plugins, and relationships to be used in your blueprints.
 
-## Inline Types
+#### Inline Types
 Inline types derive from existing types, and can define additional properties and interfaces to those available with the parent type. For each interface operations can be defined and their implementation can be selected from the available plugins.
 
-## Plugins
+#### Plugins
 Plugins can be added in the relevant tab using either a url or the implementation file.
```

Comment:
changed to:
Plugins can be added in the relevant tab using either a url (which points to a plugin.yaml or plugin.zip file) or using the plugin implementation file itself (uploading a local plugin.yaml or plugin.zip file).